src/mutable_transform_publisher_node.cpp

```diff
@@ -42,16 +43,17 @@ int main(int argc, char** argv)
   ros::NodeHandle nh, pnh ("~");
 
   std::string yaml_path;
+  double period;
   const bool yaml_specified = pnh.getParam("yaml_path", yaml_path);
-
   const bool commit = pnh.param<bool>("commit", true);
+  const bool period_specified = pnh.param<double>("period", period, 1.0);
 
   // Create the publisher
   mutable_transform_publisher::MutableTransformPublisher pub (nh);
 
-  if (yaml_specified)
+  if (yaml_specified && period_specified)
```

Comment:
Since you give a default value, is there a reason you are checking if it existed on the param server?

Reply:
Yeah, doing both is probably unnecessary. Do you think it's better to just check if it was on the parameter server or stick with the default value?

Reply:
I would just have a default value.